templates/webapi/admin/group/group_property_editor.html.ep

```diff
@@ -83,15 +83,21 @@
                 <div class="mb-3 row">
                     <label for="editor-build-version-sort" class="form-label col-sm-2 control-label" data-bs-toggle="tooltip" title="Sets how builds are sorted when groups of them are displayed together">How to sort builds for display</label>
                     <div class="col-sm-10" id="editor-build-version-sort">
-                        <input type="radio" id="editor-build-version-sort-yes" name="build_version_sort" value="1"
-                            %= 'checked' if $group->build_version_sort
+                        <input type="radio" id="editor-build-version-sort-name" name="build_version_sort" value="1"
+                            %= 'checked' if $group->build_version_sort == 1
                             >
-                        <label class="form-label" for="editor-build-version-sort-yes" data-bs-toggle="tooltip" title="Treat the build values as version numbers and sort based on those">Sort by build value (as a version)</label>
+                        <label class="form-label" for="editor-build-version-sort-name" data-bs-toggle="tooltip" title="Treat the build values as version numbers and sort based on those">Sort by build value (as a version)</label>
                         <br>
-                        <input type="radio" id="editor-build-version-sort-no" name="build_version_sort" value="0"
-                            %= 'checked' if !$group->build_version_sort
+                        <input type="radio" id="editor-build-version-sort-recent" name="build_version_sort" value="0"
```

Comment:
"recent" should be replaced with "latest" or "newest" in consistency with how you decide on the wording in controller code.